plugins/modules/jenkins_node.py

```diff
@@ -75,6 +75,62 @@
         possible. In this case, a warning will be issued.
     type: str
     version_added: 10.0.0
+  launch_ssh:
+    description:
+      - When specified, sets the Jenkins node to launch by SSH.
+    type: dict
+    suboptions:
+      host:
+        description:
+          - When specified, sets the SSH host name.
+        type: str
+      port:
+        description:
+          - When specified, sets the SSH port number.
+        type: int
+      credentials_id:
+        description:
+          - When specified, sets the Jenkins credential used for SSH authentication by
+            its ID.
+        type: str
+      host_key_verify_none:
+        description:
+          - When set, sets the SSH host key non-verifying strategy.
+        type: bool
+        choices:
+          - True
+      host_key_verify_known_hosts:
+        description:
+          - When set, sets the SSH host key known hosts file verification strategy.
+        type: bool
+        choices:
+          - True
+      host_key_verify_provided:
+        description:
+          - When specified, sets the SSH host key manually provided verification strategy.
+        type: dict
+        suboptions:
+          algorithm:
+            description:
+              - Key type e.g. V(ssh-rsa).
```

Comment:
```suggestion
              - Key type, for example V(ssh-rsa).
```
https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#avoid-using-latin-phrases